packages/server/src/integrations/base/sql.ts

```diff
@@ -89,13 +89,26 @@ function parseFilters(filters: SearchFilters | undefined): SearchFilters {
 
 function generateSelectStatement(
   json: QueryJson,
-  knex: Knex
-): (string | Knex.Raw)[] {
+  knex: Knex,
+  excludeJoinColumns = false
```

Comment:
I think we should use `opts` here:
```ts
opts: { excludeJoinColumns: boolean } = { excludeJoinColumns: false }
```

So that the usage of the function is easier to read and reason about. 
```ts
generateSelectStatement(json, knex, true) // before
generateSelectStatement(json, knex,  { excludeJoinColumns: true }) // after
```